pubsub/nats/nats.go

```diff
@@ -11,8 +11,8 @@ import (
 	"fmt"
 
 	"github.com/dapr/components-contrib/pubsub"
-	"github.com/dapr/dapr/pkg/logger"
-	nats "github.com/nats-io/go-nats"
+	"github.com/dapr/kit/logger"
+	nats "github.com/nats-io/nats.go"
```

Comment:
Why is this different?

Reply:
IIRC, I ran into errors when running `go mod tidy` that pointed back to using the old NATS client. Switching to the new client resolved the issue.